### ISSUE TITLE
Environment  map refraction not working

### ISSUE THREAD

**Issue by SamuelVanEgmond** [aframevr/aframe#5016]
**Description:**

- A-Frame Version: 1.3.0
- Platform / Device: All
- Reproducible Code Snippet or URL: https://glitch.com/~aframe-envmap

![image](https://user-images.githubusercontent.com/95985602/156890269-8e4d001e-b48c-47aa-a77e-20ca5f4934d9.png)

In the example the top sphere is supposed to be refracting, but instead it is reflecting. If I change 1.3.0 to 1.2.0 everything works.

Note: the washed out colors in the rest of the scene seem related, as that is also a result of the environment map.
However, I'm 95% sure this is same problem as described in #4989 from @dmarcos. That issue seemed to link the problem to the new scene reflection component by @AdaRoseCannon whereas in my case I use a separate equirectangular scene environment.  
Once that problem is fixed I expect the washed out colors to return to normal.

**Comment by SamuelVanEgmond** [comment]
I have looked for a solution for days before entering this issue, and posted questions on slack, but did not find anything. 
I now found a number of issues around this in https://github.com/mrdoob/three.js.  

Although refractionRatio is still in the MeshStandardMaterial in the three.js docs, there is some discussion about it.
https://github.com/mrdoob/three.js/pull/19623 seems to indicate that only Phong and Lambert materials properly support refraction. 

This means I can solve my problem with a Phong material. Even if there is a problem it is probably a three.js problem.  and not aframe. I'll therefore close the issue to not distract anyone from more important issues.

Thanks!
